src/components/file-input/UIFileInput.vue

```diff
@@ -50,7 +50,9 @@
                             'py-[1.3125rem] px-6': large,
                             'px-4 py-[0.938rem]': !large
                         }">
-                    Choose file
+                    <slot name="button-text">
```

Comment:
can you make this available as a prop too?

Reply:
Don't use slot, just prop for component?

Reply:
no, I mean both.
Simple use cases can use `<UIFileInput button-text="Choose a picture" />`
Comlex use cases eg dynamic name cn use the slot